geode-core/src/test/java/org/apache/geode/internal/cache/tx/RemoteOperationMessageTest.java

```diff
@@ -302,6 +303,52 @@ public void processWithNullPointerExceptionFromOperationOnRegionWithSystemFailur
         .hasMessageContaining("system failure");
   }
 
+
+  @Test
+  public void processInvokesDoRemoteOperationIfConserveSocketsIsFalse() {
+    doReturn("false").when(msg).getConserveSocketsSetting(dm);
+    doNothing().when(msg).doRemoteOperation(dm, cache);
+
+    msg.process(dm);
+
+    verify(msg).doRemoteOperation(dm, cache);
+    verify(msg, never()).isTransactional();
+  }
+
+  @Test
+  public void processInvokesDoRemoteOperationIfConserveSocketsIsTrueAndNotTransactional() {
+    doReturn("true").when(msg).getConserveSocketsSetting(dm);
+    doReturn(false).when(msg).isTransactional();
+    doNothing().when(msg).doRemoteOperation(dm, cache);
+
+    msg.process(dm);
+
+    verify(msg).doRemoteOperation(dm, cache);
+    verify(msg).isTransactional();
+  }
+
+  @Test
+  public void isTransactionalReturnsFalseIfTXUniqueIdIsNOTX() {
+    assertThat(msg.getTXUniqId()).isEqualTo(TXManagerImpl.NOTX);
+    assertThat(msg.isTransactional()).isFalse();
+  }
+
+  @Test
+  public void isTransactionalReturnsTrueIfCannotParticipateInTransaction() {
```

Comment:
This test name should be "isTransactionalReturnsFalseIfCannotParticipateInTransaction"